### PR TITLE
support redis ssl in feast-serving

### PR DESCRIPTION
**Pull request by oavdeev**
**What this PR does / why we need it**:
Support connecting to redis with ssl in feast-serving

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: protos/feast/core/Store.proto [hunk 1]
```diff
@@ -110,6 +110,8 @@ message Store {
     int32 max_retries = 4;
     // Optional. How often flush data to redis
     int32 flush_frequency_seconds = 5;
+
+    bool ssl = 6;
```
Comment (woop) [inline protos/feast/core/Store.proto:114]: You are missing a comment, and should this be `enable_ssl`?

Comment (oavdeev) [reply protos/feast/core/Store.proto:114]: Added a comment. This parameter is called just `ssl`/`setSsl` both in python and java client so I left it that way for consistency

**Review by woop — APPROVED** [review]

**Pull request merged**